gui/qt/main_window.py

```diff
@@ -912,19 +912,24 @@ def create_receive_tab(self):
         self.receive_address_e.addCopyButton(self.app)
         self.receive_address_e.setReadOnly(True)
         msg = _('Bitcoin Cash address where the payment should be received. Note that each payment request uses a different Bitcoin Cash address.')
-        self.receive_address_label = HelpLabel(_('Receiving address'), msg)
```

Comment:
Heh. Yeah this isn't referred to ever again. Might as well make it go away from a property.